pom.xml

```diff
@@ -43,6 +53,45 @@
                     <release>22</release>
```

Comment:
I would only use LTS java versions. Maybe matching Nano's version 21. Also define `target` and `source`